components/micro-gateway-cli/src/main/resources/templates/main.mustache

```diff
@@ -1,6 +1,7 @@
 import wso2/gateway;
 
 public function main() {
+    gateway:invokeCallHome();
```

Comment:
Lets make it a async function call

Reply:
inside callHome function it is an asynchronous execution.  so do we need make it agian ?  here I added printmessage insde this. but it is not async

Reply:
fixed